PaymentHighway/Extensions.swift

```diff
@@ -40,17 +39,17 @@ internal extension String {
     /// - returns: Found matches as an array
     func matchesForRegex(_ regex: String!) -> [String] {
         
-        let regex = try! NSRegularExpression(pattern: regex,
-            options: [])
+        guard let regex = try? NSRegularExpression(pattern: regex, options: []) else { return [] }
```

Comment:
Should this `throw` instead of silently failing for invalid expressions?

Reply:
A lot of unwrapping optional were used in the code . The point here is to make the code not crashing.
Change of the function interface might be possible of course but should not be part of the lint fix IMO.